Services/GEV/DecentralTrainings/classes/class.gevDecentralTrainingCreationRequest.php

```diff
@@ -456,4 +459,33 @@ protected function updateCourseWithBuidlingBlockData($a_trgt_crs_ref_id) {
 		require_once("Services/GEV/Utils/classes/class.gevCourseBuildingBlockUtils.php");
 		gevCourseBuildingBlockUtils::courseUpdates($a_trgt_crs_ref_id);
 	}
+
+	protected function addAttachmentsToMail($trgt_obj_id) {
+		require_once("Services/GEV/DecentralTrainings/classes/class.gevDecentralTrainingFileStorage.php");
```

Comment:
Bitte die Erzeugung des FileStorage in eine eigene Methode auslagern, analog zu getDecentralTrainingUtils etc.